make.py

```diff
@@ -96,9 +96,11 @@
 
 
 @cli
-async def process(itl_path: Path=Path('boundary.json'),
-                  disputed_path: Path=Path('disputed.json'),
+async def process(itl_path: Path=Path('exports/boundary.json'),
+                  disputed_path: Path=Path('exports/disputed.json'),
                   database='mae'):
+    exports = Path('exports')
```

Comment:
You can do something like:

    `itl_path.parent.mkdir(…)`

Reply:
Ok sure, it's replaced